pyshp/bld.bat

```diff
@@ -1 +1,2 @@
-%PYTHON% setup.py install
\ No newline at end of file
+%PYTHON% setup.py install install --single-version-externally-managed --record record.txt
```

Comment:
Good no!  My adolescent days of late night coding are over!! 

I am GMT-3 (Northeast Brazil).  GitHub says that commit was ~17:30 and that is late for my age :wink: